qiskit/test/base.py

```diff
@@ -227,7 +227,7 @@ def setUpClass(cls):
             r"elementwise comparison failed.*",
             r"The jsonschema validation included in qiskit-terra.*",
             r"The DerivativeBase.parameter_expression_grad method.*",
-            r"'Bit\.(register|index)' is deprecated.*",
+            r"The property ``qiskit\.circuit\.bit\.Bit\.(register|index)`` is deprecated.*",
```

Comment:
this is only semi-related: the error message we've got here shows a potential place for improvement in the decorators to me; `qiskit.circuit.bit.Bit.register` feels way too long to put out.  I think `Bit.register` would be less noisy - the user will always be able to inspect the object themselves if they need to know more about it.